tests/core/dom/range/gettableelement.js

```diff
@@ -42,11 +42,11 @@
 		return range;
 	}
 
-	function assertElements( expected, range, selectors ) {
+	function assertElements( expected, range, selectors, message ) {
```

Comment:
TBH when I see function with more than three arguments, I always think that it should be replaced simply by `options`:

```javascript
function assertElement( options )
```

Reply:
I've changed it a little bit different. As `expected` and `range` are required parameters and has to be provided in every assertion I remain it without change. Those are not options form my perspective, and that's why should be hidden inside option object. Other parameters are moved to the options. 
```
function assertElement( expected, range, options )
```

I also add little docs describing parameters.